steward/src/utils.rs

```diff
@@ -88,3 +99,49 @@ pub async fn get_eth_provider() -> Result<Provider<Http>, Error> {
 pub fn sp_call_error(message: String) -> Error {
     ErrorKind::SPCallError.context(message).into()
 }
+
+pub struct SubmitCork {
```

Comment:
Rather than creating another type and implementing a function on it, we could simply define a `submit_scheduled_cork` function that takes a `Cork` and a height as arguments.